src/test/unit_tests/test_bucketspace.js

```diff
@@ -3,43 +3,37 @@
 /*eslint max-statements: ["error", 80, { "ignoreTopLevelFunctions": true }]*/
 'use strict';
 
-
-const mocha = require('mocha');
+const fs = require('fs');
+const os = require('os');
+const _ = require('lodash');
+const path = require('path');
 const util = require('util');
+const http = require('http');
+const mocha = require('mocha');
 const AWS = require('aws-sdk'); // GAP upload is still using AWS SDK V2
+const assert = require('assert');
+const config = require('../../../config');
 const { S3 } = require('@aws-sdk/client-s3');
```

Comment:
I guess you also did some order in this part, would you please leave those in sequence (one after another):
```
const AWS = require('aws-sdk'); // GAP upload is still using AWS SDK V2
const { S3 } = require('@aws-sdk/client-s3');
const { NodeHttpHandler } = require("@smithy/node-http-handler");
```

Reply:
np, done